sakai-evaluation-tool/src/webapp/content/templates/modify_hierarchy_node_rules.html

```diff
@@ -0,0 +1,140 @@
+<!--
+
+    Copyright 2005 Sakai Foundation Licensed under the
+    Educational Community License, Version 2.0 (the "License"); you may
+    not use this file except in compliance with the License. You may
+    obtain a copy of the License at
+
+    http://www.osedu.org/licenses/ECL-2.0
+
+    Unless required by applicable law or agreed to in writing,
+    software distributed under the License is distributed on an "AS IS"
+    BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express
+    or implied. See the License for the specific language governing
+    permissions and limitations under the License.
+
+-->
+<!DOCTYPE html PUBLIC "-//W3C//DTD XHTML 1.0 Transitional//EN" "http://www.w3.org/TR/xhtml1/DTD/xhtml1-transitional.dtd">
+<html xmlns:rsf="http://ponder.org.uk/rsf" xmlns="http://www.w3.org/1999/xhtml">
+    <head>
+        <title rsf:id="msg=modifynoderules.page.title">Hierarchy Node Rules</title>
+
+        <!-- standard on all pages -->
+        <meta http-equiv="Content-Type" content="text/html; charset=UTF-8" />
+        <link href="/library/skin/tool_base.css" rsf:id="scr=portal-matter" type="text/css" rel="stylesheet" media="all"/>
+        <link href="/library/skin/default/tool.css" rsf:id="scr=portal-matter" type="text/css" rel="stylesheet" media="all"/>
+        <script src="/library/js/headscripts.js" rsf:id="scr=portal-matter" type="text/javascript" language="JavaScript"></script>
+        <!-- end standard on all pages -->
+
+        <script type="text/javascript" language="JavaScript" src="../js/jquery/jquery-1.4.4.min.js"></script>
+        <script type="text/javascript" language="JavaScript" src="../js/rsf.js"></script>
+        <script type="text/javascript" language="JavaScript" src="../js/evalUtils.js"></script>
+
+        <link href="../css/evaluation_base.css" type="text/css" rel="stylesheet" media="all" />
+        <link rsf:id="local_css_include" href="local_css_file.css" type="text/css" rel="stylesheet" media="all" />
+    </head>
+
+    <body>
+        <div class="portletBody">
+            <div rsf:id="navIntraTool:" class="navIntraTool"/>
+
+            <ul class="breadCrumb" style="clear: both; height: 1.5em">
```

Comment:
Same as above comment; should likely be addressed in a separate JIRA project wide.